### DISCUSSION TITLE
jwt role with special characters

### DISCUSSION THREAD

**Discussion by bmgante** [PostgREST/postgrest#2281] [Q&A] [answered]
Is there any way to apply some regex or similar to the role-claim-key in order to return only a substring of the value? To address a use case where the role in the claim contains special characters such as "-" and "." which are not allowed in postgres roles, ending up with no match possible between roles.

**Comment by steve-chavez** [comment, marked as answer]
> To address a use case where the role in the claim contains special characters such as "-" and "." which are not allowed in postgres roles,

You can use those special characters in a pg role:

```
postgres=# create role "look-at.myrole";
CREATE ROLE
```

Are you seeing an error when trying that type of role through PostgREST?

**Reply by bmgante** [reply]
Indeed, using double quotes it works, perfect :) Thanks
